### PR TITLE
Refactor empty state and rearrange components routes alphabetical

### PR DESCRIPTION
**Pull request by mkumbobeaty**

### CODE REVIEW

File: src/layouts/BaseLayout.js [hunk 1]
```diff
@@ -53,49 +53,140 @@ import GeographicalFeaturesInfrastructureLayout from '../GeographicalFeatures/la
...
+  },
+  '/assessments/questions': {
+    name: 'Questions',
+    title: 'List of questions',
+  },
+  '/assessments/responses': {
+    name: 'Responses & Observations',
+    title: 'Assissment Responses and observations',
+  },
+  /* end of assissment components */
```
Comment (BenMaruchu) [inline src/layouts/BaseLayout.js]: there is a typo here

Comment (BenMaruchu) [reply src/layouts/BaseLayout.js]: Also I have a suggestion 

we can use separators like this
```js
/* Assessments routes */
  '/assessments': { name: 'Assessments', title: 'Assessments module' },
  '/assessments/indicators': {
    name: 'Indicators',
    title: 'Indicators for monitoring assissment',
  },
  '/assessments/questionnaires': {
    name: 'Questionnaires',
    title: 'List of questionnaires',
  },
  '/assessments/questions': {
    name: 'Questions',
    title: 'List of questions',
  },
  '/assessments/responses': {
    name: 'Responses & Observations',
    title: 'Assissment Responses and observations',
  },

/* Geographical Routes */
...
```

let remove `end of something`  line

Comment (BenMaruchu) [reply src/layouts/BaseLayout.js]: Also there are many typos in this file ... 
it is supposed to be `assessments` and not `assissment`

Comment (BenMaruchu) [reply src/layouts/BaseLayout.js]: Also on the separation comments please use `routes` and not `components` since these are routes

**Review by BenMaruchu — CHANGES_REQUESTED** [review]
Hey @mkumbobeaty  Nice work. There are some few things needs updating, if you don't understand it please ask.

**Comment by BenMaruchu** [comment]
@mkumbobeaty  can you pull changes from develop and fix if there is any issues cause I can see conflicts here

**Comment by mkumbobeaty** [comment]
Oky

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**